docs/introduction/getting-started.md

```diff
@@ -95,6 +95,8 @@ RTK Query includes these APIs:
 
 - [`createApi()`](../rtk-query/api/createApi.mdx): The core of RTK Query's functionality. It allows you to define a set of endpoints describe how to retrieve data from a series of endpoints, including configuration of how to fetch and transform that data.
 - [`fetchBaseQuery()`](../rtk-query/api/fetchBaseQuery.mdx): A small wrapper around [`fetch`](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) that aims to simplify requests. Intended as the recommended `baseQuery` to be used in `createApi` for the majority of users.
+- [`createApi()`](../rtk-query/api/createApi.mdx): The core of RTK Query's functionality. It allows you to define a set of endpoints describe how to retrieve data from a series of endpoints, including configuration of how to fetch and transform that data. In most cases, you should use this once per app, with "one API slice per base URL" as a rule of thumb.
```

Comment:
Is the duplication here intended or should that go into the previous paragraphs?

Reply:
🤦 

Not intended, just a dodgy conflict resolution when re-basing on my behalf. Should be better now

Reply:
Good I caught it then. Apart from that, everything looked fine, so :+1: